src/components/dataTile/index.tsx

```diff
@@ -0,0 +1,26 @@
+import './DataTile.scss';
+
+type Props = {
+  children: string;
+  title?: string;
+  theme?: 'blue' | 'green' | 'teal' | 'yellow';
+};
+
+const DataTile = (props: Props) => {
+  const {children, title, theme} = props;
+  const tileTheme = theme ?? 'blue';
+  const defaultTitle = {
+    blue: 'Aktualna temperatura',
+    green: 'Aktualne ph wody',
+    teal: 'Status światła',
+    yellow: 'Status grzałki',
+  };
+  return (
+    <div className={`tile tile-${tileTheme.toLowerCase()}`}>
+      <p>{title ?? defaultTitle[tileTheme]}</p>
+      <p>{children}</p>
+    </div>
+  );
+};
+
+export default DataTile;
```

Comment:
![image](https://user-images.githubusercontent.com/95183973/225715616-541fdd20-0fe8-493f-b491-28f933c4730f.png)
zrób reeksport komponentu z innego pliku
plik **index.ts** umieść wewnątrz katalogu components (jeśli go tam nie ma), a następnie zrób taki eksport jak na zdjęciu wyżej dla twojego komponentu. Wówczas import z pliku working/index.tsx powinien wyglądać mniej więcej tak:
import { DataTile } from "../../components"
fajnie by było jakbyś zrobił to samo dla wszystkich innych komponentów, które są już w projekcie

Reply:
> plik index.ts umieść wewnątrz katalogu components

w sensie `components` czy `components/dataTile`?
Bo nie wiem czy chodzi o to by utworzyć jeden plik `index.tx` w katalogu `components` i wsadzić tam exporty wszystkich komponentów, czy do każdego katalogu komponentowego (w którym jeszcze nie ma) dodać `index.ts` osobno